### ISSUE TITLE
PLZ，Is it normal that the accuracy is particularly low on my own classification data set？

### ISSUE THREAD

**Issue by BJFU-CS2012** [lblaoke/TLC#8]
I have successfully run your project, but when I use my own dataset, the accuracy is only about 10%. What do you think are the possible reasons? Thank you

**Comment by BJFU-CS2012** [comment]
about 18 classes ,using 4310 images for training, 1068 images for validation.

**Comment by lblaoke** [comment]
You can simplify the objective to cross entropy to see if the loss is going down at training. This will help debugging

**Comment by BJFU-CS2012** [comment]
Thank you again for your reply. Let me test it

**Comment by IABAcademy** [comment]
I met a similar problem. 
As long as I'm concerned, there are three main components in your paper:
1. Loss for single expert; which is basically an nll_loss and a kl 
2. Loss for kl (for multi experts)
3. w(for multi experts)
When I use component 1 on my own dataset(to train a single expert as baseline), the Recall is almost the same as using cross entropy. However, the mean Recall(a metric used to evaluate long tail classification ability) is even worse than using simple cross entropy. 
I want to know: if I use cross entropy to train each single expert, instead of component 1, and still use component 2 and 3, would I still get bonus in long tail classification ability?
If you have found anything new, I look forward to your reply. Thank you for your time.

**Comment by lblaoke** [comment]
You may consider using a re-weighted loss (component 1) if your dataset is also long-tailed. Specifically, the loss values should be larger for smaller classes. This is a widely used trick in imbalanced learning and you can also find it in this repository (model/loss.py).

**Comment by IABAcademy** [comment]
Thank you very much! I'll try on the re-weighted loss to see if I can get normal results.
